### DISCUSSION TITLE
ks.concat indexing when ignore_index=True

### DISCUSSION THREAD

**Discussion by barashe** [databricks/koalas#2214] [Q&A] [answered]
From the Koalas [docs](https://koalas.readthedocs.io/en/latest/reference/api/databricks.koalas.concat.html):

> ignore_index : bool, default False
> - If True, do not use the index values along the concatenation axis. The resulting axis will be labeled 0, …, n - 1.

So, if `ignore_index=True` the indexing process ignores `compute.default_index_type`? For example, if I assign `distributed` to `compute.default_index_type`, will this still be a `sequence` indexing process?

**Comment by itholic** [comment, marked as answer]
No, "ignoring index" and "adding default index" is different.

`ignore_index=True` just ignores the existing index, and attach the new default index as defined in `compute.default_index_type` (so the index order could be mixed, but it still "ignores" the existing index)

If you want to get a fully sequenced index, you should set the `compute.default_index_type` to `sequence`.
